core/src/components/datetime/datetime.tsx

```diff
@@ -153,6 +153,22 @@ export class Datetime implements ComponentInterface {
    */
   @Prop() readonly = false;
 
+  /**
+   * Returns if an individual date (calendar day) is enabled or disabled.
+   *
+   * If `true`, the day will be enabled/interactive.
+   * If `false`, the day will be disabled/non-interactive.
+   *
+   * The function accepts an ISO 8601 date string of a given day.
+   * By default, all days are enabled. Developers can use this function
+   * to write custom logic to disable certain days.
+   *
+   * Custom implementations should be optimized for performance. This
+   * function is called often, so any extra logic should be avoided
+   * to reduce and prevent jank.
```

Comment:
```suggestion
   * The function is called for each rendered calendar day, for the previous, current and next month.
   * Custom implementations should be optimized for performance to avoid jank.
```

Reply:
Non-blocking, but this matches the docs below